xml/System.Security.Cryptography/IncrementalHash.xml

```diff
@@ -54,8 +54,8 @@
         <ReturnType>System.Security.Cryptography.HashAlgorithmName</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the name of the algorithm being performed.</summary>
+        <value>The name of the algorithm being performed.</value>
```

Comment:
Ok, I've added that to the summary. That behavior might be easier to show in a sample.